daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DFDLDelimiter.scala

```diff
@@ -445,7 +445,7 @@ class CharDelim(val char: Char, ignoreCase: Boolean) extends DelimBase {
   }
 
   lazy val typeName = "CharDelim"
-  def print = {
+  def print() = {
```

Comment:
Dead code - left over from debugging. Remove or comment out fully? Ditto for all other print() methods in this file.